app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/firestore/DatasetsManagement.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.HumanLearning2021.HumanLearningApp.firestore
```

Comment:
```suggestion
package com.github.HumanLearning2021.HumanLearningApp.model
```

Even if FS is the only implementation, the interface itself can be kept generic